unicodetools/data/linkification/dev/LinkEmail.txt

```diff
@@ -0,0 +1,1298 @@
+# LinkEmail.txt
+# Date: 2025-12-14, 06:39:23 GMT 
+# © 2025 Unicode®, Inc.
+# Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+# For terms of use and license, see https://www.unicode.org/terms_of_use.html
+#
+# The usage and stability of these values is covered in https://www.unicode.org/reports/tr58/
+#
+# ================================================
+#
+# Property:	Link_Email
+# Format
+#
+# Field 0: code point range
+# Field 1: binary value
```

Comment:
I guess this works, but it looks a bit odd. The second column is basically useless.

This must be the first time that we have a data file dedicated to one single **binary** property.

When we list binary properties in multi-property data files, we don't have a value column. By analogy, we only really need a single code point column here.